src/System.Private.CoreLib/shared/System/Globalization/CultureNotFoundException.cs

```diff
@@ -10,7 +11,7 @@ namespace System.Globalization
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public class CultureNotFoundException : ArgumentException
     {
-        private string _invalidCultureName; // unrecognized culture name
+        private string? _invalidCultureName; // unrecognized culture name
         private int? _invalidCultureId;     // unrecognized culture Lcid
 
         public CultureNotFoundException()
```

Comment:
All of these `string` and `Exception` ctor arguments should be nullable.

Reply:
Yes, I annotated Exception and ArgumentException yesterday, I wonder why you're not getting errors, maybe you haven't absorbed my changes here?